packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -32,7 +32,7 @@ const getModulePath = (version: string) => path.join(baseModulePath, version);
 const resolutionCache = new Map<string, string>();
 
 // Increment the revision if we want to force installation (e.g. package list changed)
-const revision = 1;
+const revision = 2;
```

Comment:
When merging this change over main, I encountered incremental build issues because the set of packages to install for compat purposes changed. Though it's likely most people have already paid this cost, I figured I might as well bump the revision here in case anyone has some stale repos that they start using again without doing a full reinstall of node_modules.